lecture_1/cli_args.js

```diff
@@ -1,26 +1,31 @@
-const fs = require("fs");
 const { stat } = require("fs/promises");
+var argv = require('minimist')(process.argv.slice(2));
 
-const filePath = process.argv[2];
+if (!argv['path'] || typeof argv['path'] === 'boolean') {
+  console.log('--path parameter is required');
+  process.exit();
```

Comment:
It's better to use non-zero exit code on error exit

Reply:
btw, process.exit() is not recommended practice
https://nodejs.org/api/process.html#process_process_exit_code